src/builtins/builtins-wasm-gen.cc

```diff
@@ -110,6 +111,46 @@ TF_BUILTIN(WasmGrowMemory, WasmBuiltinsAssembler) {
   ReturnRaw(Int32Constant(-1));
 }
 
+TF_BUILTIN(WasmToBigIntJavaScript_WasmRuntimeTrampoline,
+           WasmBuiltinsAssembler) {
+  TNode<Object> context = UncheckedParameter(Descriptor::kContext);
+  TNode<Object> argument = UncheckedParameter(Descriptor::kArgument);
+  TNode<Code> target = LoadBuiltinFromFrame(Builtins::kWasmToBigIntJavaScript);
+  TailCallStub(TypeConversionDescriptor(), target, context, argument);
+}
+
+TF_BUILTIN(WasmToBigIntJavaScript, WasmBuiltinsAssembler) {
+  TNode<BigInt> value_bigint;
+
+  TNode<IntPtrT> ptr =
+      UncheckedCast<IntPtrT>(Parameter(Descriptor::kArgument));
```

Comment:
Hmm, I'm not sure if it's right to just pass a bare pointer to a 64-bit integer as a parameter to a built-in. This wouldn't confuse the GC?

Reply:
The pointer type is actually used for holding the int64 value (or an int32 on 32 bits arch). It's consistent with the `BigIntFromInt64` function.

Reply:
I think BigIntFromInt64 is a little different, since it's just a stub that gets generated inline, and isn't a call according to V8's calling conventions.

Reply:
Agreed, i'm working on it.

Reply:
This is actually poiting to a BigInt Object, not a raw int64.